web/src/emoji_frequency.ts

```diff
@@ -0,0 +1,174 @@
+import assert from "minimalistic-assert";
+
+import * as all_messages_data from "./all_messages_data.ts";
+import * as emoji_picker from "./emoji_picker.ts";
+import * as message_store from "./message_store.ts";
+import * as reactions from "./reactions.ts";
+import {current_user} from "./state_data.ts";
+import * as typeahead from "./typeahead.ts";
+
+export type ReactionData = {
```

Comment:
Should we call this `ReactionUsage`? You're using `reaction_usage` as the local variable, and I think the more specific type name would be helpful.